.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+<Update pull_request_template.md to change this template>
+<Delete any sections below that are not applicable>
+
+# Description
+
+Issue: #<issue number if applicable>
+
+- [ ] Change 1
+- [ ] Change 2
+- [ ] Change 3
+
+# Screenshots
+
+<details>
+ <summary>Click here for screenshots</summary>
+
+[Add screenshots here]
```

Comment:
Might make sense to make this directive a comment as well to avoid the extra deletion when adding screenshots :)

```suggestion
<Add screenshots here>
```

Reply:
Makes sense, updated the template in [this commit](https://github.com/mitxpro-dev/learnjs/pull/7/commits/e4a593e6699ede3f866366fecdd23830b7eff7a7).